docs/tensor/movement.md

```diff
@@ -7,6 +7,7 @@
 ::: tinygrad.Tensor.flip
 ::: tinygrad.Tensor.shrink
 ::: tinygrad.Tensor.pad
+::: tinygrad.Tensor.rearrange
```

Comment:
it should not be here

Reply:
my bad, where is the best place to put it?